rails/app/controllers/api/analytics_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::AnalyticsController < ApplicationController
+  before_filter :authorize_user, only: [:index]
```

Comment:
- [ ] TODO: Yeah, I see that. Regardless I want this entire file to be protected so people aren't using my Google Analytics API